packages/mobile/src/utils/currencies.ts

```diff
@@ -0,0 +1,35 @@
+export enum Currency {
+  Celo = 'cGLD',
+  Dollar = 'cUSD',
+  Euro = 'cEUR',
+}
```

Comment:
Maybe for another PR:
```suggestion
export enum CurrencyCode {
  CELO = 'CELO',
  cUSD = 'cUSD',
  cEUR = 'cEUR',
}
```